public/js/index.js

```diff
@@ -216,6 +216,35 @@ function initBranchSelector() {
     });
 }
 
+function initRepoStatusChecker() {
+    console.log("initRepoStatusChecker")
+    var migrating = $("#reop_migrating");
+    if (migrating) {
+        var repo_name = migrating.attr('repo');
+        if (repo_name == undefined) {
```

Comment:
Do not use `undefined`. Better to use `typeof repo_nane === 'undefined'`

Reply:
done.